contracts/voting-verifier/src/contract/execute.rs

```diff
@@ -95,6 +98,14 @@ pub fn verify_messages(
 
     let config = CONFIG.load(deps.storage).expect("failed to load config");
 
+    // Error in case we have messages to ITS Hub
+    if let Some(_) = messages.iter().find(|msg| {
+        msg.destination_chain == AXELAR_CHAIN_NAME
+            && msg.destination_address.as_str() == config.its_hub_address.as_str()
+    }) {
+        return Err(ContractError::InvalidMessages.into());
+    }
```

Comment:
this guard makes me a little nervous. If the ITS hub address ever changes, or the axelar chain name ever changes, we will have two different representations of the same message, which could cause a serious problem. It's highly unlikely those things will ever change, but this is too thin of a protection around something that is very critical (representations of the same message). I think since we plan to move all of this translation logic into the hub, I would prefer we just remove the standard `VerifyMessages` function for now, and just restrict all messages to be ITS messages.

Reply:
without this condition, a given message with a given cross chain id could be routed with two different payloads. once with the clarity payload, and once with the abi payload. This would be a huge issue. Yes, this check prevents that, but this is a very thin guard around a big security issue. What if it's instantiated wrong, if even for a brief period? I just don't like being so close to a major issue, and as far as I understand, there are no other apps besides ITS planned for the launch. given that, i think it's much cleaner and safer to just remove this.

Reply:
Similar to the prover comment, if it's complicated to remove this method from the existing codebase, I recommend only allowing governance to execute this. And after ITS hub translation is supported, we can open this up

Reply:
why remove the functionality in the prover but make it only callable by governance in the verifier?